src/model_loader_tinygltf.cpp

```diff
@@ -0,0 +1,291 @@
+#include "model_loader_tinygltf.hpp"
+
+#include "util/log.hpp"
+
+#define TINYGLTF_IMPLEMENTATION
+#define STB_IMAGE_IMPLEMENTATION
+#define STB_IMAGE_WRITE_IMPLEMENTATION
+#include <tiny_gltf.h>
+
+namespace wr
+{
+
+	TinyGLTFModelLoader::TinyGLTFModelLoader()
+	{
+		m_supported_model_formats = { "gltf" };
+	}
+
+	TinyGLTFModelLoader::~TinyGLTFModelLoader()
+	{
+	}
+
+	std::pair<std::vector<DirectX::XMFLOAT3>, std::vector<DirectX::XMFLOAT3>> ComputeTangents(ModelMeshData* mesh_data)
+	{
+		size_t vtxCount = mesh_data->m_positions.size();
+		std::vector<DirectX::XMFLOAT3> tanA(vtxCount, { 0, 0, 0 });
+		std::vector<DirectX::XMFLOAT3> tanB(vtxCount, { 0, 0, 0 });
+
+		if (mesh_data->m_uvw.empty())
+		{
+			return { tanA, tanB };
+		}
+
+		// (1)
+		size_t indexCount = mesh_data->m_indices.size();
+		for (size_t i = 0; i < mesh_data->m_indices.size(); i += 3) {
```

Comment:
According to the style guide, a bracket should be on a new line.

Reply:
done